src/corppa/ocr/gvision_ocr.py

```diff
@@ -1,66 +1,293 @@
 #!/usr/bin/env python
 
 """
-setup:
- - in google cloud console, enable vision api for desired project
- - install gcloud cli (sdk) and login (this is now preferred to service accounts)
- - install python client: `pip install google-cloud-vision`
+This script OCRs image using the Google Vision API.
 """
 
-import glob
-import io
 import os
+import sys
+import io
+import pathlib
+import argparse
+
+from tqdm import tqdm
+from corppa.utils.path_utils import get_vol_dir
+
+# Attempt to import Google Cloud Vision Python Client
+try:
+    from google.cloud import vision
+except ImportError:
+    print(
+        "Error: Python environment does not contain google-cloud-vision "
+        "package. Switch environments or install package and try again."
+    )
+    sys.exit(1)
```

Comment:
we might need to move this but I think let's wait and figure it out when we look at unit testing